wastewater/dataModel.WasteWaterTreatment/Junction/schema.json

```diff
@@ -0,0 +1,75 @@
+{

```

Comment:
We have a problem with the name of this data model because it [already exists](https://github.com/smart-data-models/dataModel.WaterNetworkManagement/tree/master/Junction) a junction in waterNetwrokmanagement subject. Please review it and propose a marge (in the case is the same concept) o an alternative name. 
Bes